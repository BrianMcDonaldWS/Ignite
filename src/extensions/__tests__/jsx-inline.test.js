import MarkdownIt from 'markdown-it';
import jsx from 'markdown-it-jsx';
import inline from '../jsx-inline';

const md = new MarkdownIt().use(jsx).use(inline);

test('jsx - inline', () => {
  expect(md.render('<Foo />')).toMatchSnapshot();
});
