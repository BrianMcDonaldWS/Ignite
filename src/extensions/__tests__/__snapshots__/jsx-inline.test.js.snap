// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`jsx - inline 1`] = `
"<pluginprovider name=!{'Foo'!} plugins=!{props.plugins!} />
"
`;
